spec/controllers/books_controller_spec.rb

```diff
@@ -2,6 +2,10 @@
 require 'rails_helper'
 
 RSpec.describe BooksController do
+  let(:user) { FactoryGirl.create(:admin) }
```

Comment:
I've seen CanCan do weird stuff if the user isn't persisted. :( Fortunately it doesn't seem to hurt the runtime of the suite much.